guides/common/modules/proc_configuring-host-based-access-control-for-freeipa-users-logging-in-to-foreman.adoc

```diff
@@ -0,0 +1,119 @@
+endif::[]
+. On your {ProjectServer}, load the host-based access control rules from {FreeIPA}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-pam-service=satellite-prod
+----
+
+.Verification
+
+* Log in to the {ProjectWebUI} as a user defined in {FreeIPA}.
+** If the user is included in the HBAC rule, {ProjectWebUI} will grant access.
+** If the user is not included in the HBAC rule, {ProjectWebUI} will not grant access.
+
+ifndef::satellite[]
+.Additional resources
+
+* For more information about the `allow_all` rule and configuring HBAC in {FreeIPA}, see link:https://www.freeipa.org/page/Howto/HBAC_and_allow_all[HBAC and `allow_all`] in {FreeIPA} documentation.
+
+endif::[]
+.Configuring host-based access control to allow access to {Project} only for selected {FreeIPA} users by using the command line
+====
+
+On the {FreeIPA} server, a user with administrative privileges configures a HBAC rule to allow selected users access to {ProjectServer}:
+
+. Authenticate as the user with privileges required to configure HBAC rules:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ kinit _admin_
+----
+. Optional: Verify that you have authenticated successfully:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ klist
+----
+. Create a new HBAC service named `{project-context}-prod`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ ipa hbacsvc-add {project-context}-prod
+----
+. Create a new HBAC rule:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ ipa hbacrule-add _allow-{project-context}-prod_
+----
+. Add the following {FreeIPA} entities to the HBAC rule:
+.. The `{project-context}-prod` HBAC service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ ipa hbacrule-add-service _allow-{project-context}-prod_ --hbacsvcs={project-context}-prod
+----
++
+.. The {ProjectServer} host:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ ipa hbacrule-add-host _allow-{project-context}-prod_ --hosts=_{foreman-example-com}_
+----
++
+.. The users or user groups to whom you want to grant access:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ ipa hbacrule-add-user _allow-{project-context}-prod_ --user=_ipa-user_
+----
++
+. Optional: Verify the status of the rule:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ ipa hbacrule-find _{project-context}-prod_
+$ ipa hbactest --user=_ipa-user_ --host=_{foreman-example-com}_ --service={project-context}-prod
+----
+. Disable the default `allow_all` rule:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ ipa hbacrule-disable allow_all
+----
+
+On {ProjectServer}, a {Project} administrator re-runs {foreman-installer} to load the host-based access control rules from {FreeIPA}:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-pam-service={project-context}-prod
```

Comment:
Tested on Foreman and Satellite via "foreman-installer --full-help":

````
    --foreman-pam-service                                                        PAM service used for host-based access control in IPA (current: "foreman") # Satellite
    --foreman-pam-service                                                        PAM service used for host-based access control in IPA (current: "foreman") # Foreman 3.11
````

Also affects line 105, 34, maybe more. Can you please recheck if this is branded in an upcoming Satellite release?